web/components/PaneMap.vue

```diff
@@ -8,15 +8,15 @@
         indeterminate />
     </div>
     <SearchBarLocation
-      class="search-bar-location"
-      v-model="internalLocation" />
+      v-if="!drawerOpen"
+      :class="{
```

Comment:
something about this doesn't feel right: we're adding a `drawer-open` class (with CSS below), but the condition doesn't really have to do with the drawer being open, and in any case we hide this bar from `PaneMap` altogether when the drawer is open.

can we remove this class?  please test.